templates/distribution/manifests/dr/kustomization.yaml.tpl

```diff
@@ -41,13 +45,33 @@ patchesStrategicMerge:
   - patches/velero-schedule-manifests.yml
   - patches/velero-schedule-full.yml
 {{- end }}
+{{- if and (index .spec.distribution.modules.dr "etcdBackup") (.spec.distribution.modules.dr.etcdBackup.s3.enabled) }}
+  - patches/etcd-backup-schedule.yml
+{{- end }}
 
 {{- if eq .spec.distribution.common.provider.type "none" }}
-{{- if eq .spec.distribution.modules.dr.velero.backend "externalEndpoint" }}
+{{- if and (index .spec.distribution.modules.dr "etcdBackup") (.spec.distribution.modules.dr.etcdBackup.s3.enabled) }}
+configMapGenerator:
+  - name: etcd-backup-config
+    behavior: replace
+    literals:
+      - target={{ print "minio:" .spec.distribution.modules.dr.etcdBackup.s3.bucketName }}
+      - retention={{ .spec.distribution.modules.dr.etcdBackup.s3.retentionTime }}
+{{- end }}
+{{- end }}
+
 secretGenerator:
```

Comment:
Are we sure this will not break aws provider? It's a kustomize statement without array elements in that case, If I understood correctly all the if opening and closing

Reply:
I figured that we were doing this in other places, so I though it would be fine: https://github.com/sighupio/fury-distribution/blob/97c89b5a4e1b4bef28e586234092329bf0c206b6/templates/distribution/manifests/monitoring/kustomization.yaml.tpl#L123

Reply:
Check this with EKS.

Reply:
Tested with `none` and it doesn't break.